src/main/cpp/systemds.h

```diff
@@ -29,65 +29,65 @@ extern "C" {
 /*
  * Class:     org_apache_sysds_utils_NativeHelper
  * Method:    dmmdd
- * Signature: ([D[D[DIIII)Z
+ * Signature: ([D[D[DIIII)I
```

Comment:
this docs seems wierd.
does the coding of this make a difference for the docs constructed?
otherwise i would consider using double* and so on.

Reply:
That is machine generated stuff by javah. I couldn't resist to update the signature after changing the function's parameters even though it says specifically to not edit this file. So yes, I admit that I did not bother to run through javah again for the changes I made 😊